samples/nrf9160/cloud_simple/README.rst

```diff
@@ -0,0 +1,110 @@
+.. _cloud_simple:
+
+nRF9160: Cloud Simple
+#####################
+
+Sample that connects to and communicates with a compatible cloud service using the respective firmware cloud backend. The sample connects via cellular network (LTE) and publishes a custom string in intervals or upon a button trigger, to the cloud service.
+
+Overview
+********
+
+The ``cloud_simple`` sample demonstrates how to use the generic cloud API ``subsys/net/lib/cloud`` to interface with multiple cloud backends. The current version of the ``cloud_simple`` sample supports the following libraries as cloud backends:
+
+ -  :ref:`lib_nrf_cloud`
+ -  :ref:`lib_aws_iot`
+
+To swap between cloud backends simply change the option ``CONFIG_CLOUD_BACKEND`` to match the configuration string of a compatible cloud backend. The identifying string for the different cloud backends are listed below.
+
+ - **NRF Cloud**, configuration string ->"NRF_CLOUD"
+ - **AWS IoT**, configuration string -> "AWS_IOT"
+
+Setup
+*****
+
+For setting up the different cloud backends please see the corresponding documentations. Each cloud backend has spesific setup steps that must be executed before it can be used. These setup steps cover generating and flashing *certificates* used for the cloud connection, setting up the correct connection *endpoint* etc..
+
+.. note::
+   The nRF9160 DK and Thingy:91 comes stock with certificates required for connection to NRF Cloud. For NRF Cloud no extra setup steps are required to use the ``cloud_simple`` sample. See `NRF Cloud web interface <https://nrfcloud.com/#/>`_ for more information.
+
+
+Configurations
+**************
+
+The configurations for the sample are located in ``cloud_simple/src/prj.conf``
+
+ - ``CONFIG_CLOUD_BACKEND`` <- Determines the cloud backend to be used.
+ - ``CONFIG_CLOUD_PUBLICATION_SEQUENTIAL`` <- Publish message to cloud sequentially.
+ - ``CONFIG_CLOUD_PUBLICATION_BUTTON`` <- Publish message to cloud upon a button press.
+ - ``CONFIG_CLOUD_MESSAGE`` <- Modifies the message published to the cloud service.
+ - ``CONFIG_CLOUD_MESSAGE_PUBLICATION_INTERVAL`` <- Modifies how often the message should be published to the cloud service.
+
+.. note::
+   To output data in the terminal window located in the `NRF Cloud web interface <https://nrfcloud.com/#/>`_ the data format must be in JSON format.
+
+Functionality and Supported Technologies
+****************************************
+
+The communicaton protocol supported by the sample is dependent on the cloud backend that is used. Please see the documentation for the individual cloud backend libraries.
```

Comment:
```suggestion
The communication protocol supported by the sample is dependent on the cloud backend that is used.
```
We already have links to the cloud backend documentation earlier in the documentation.